extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeMixPlaylistExtractorTest.java

```diff
@@ -50,7 +53,8 @@
 
         @BeforeClass
         public static void setUp() throws Exception {
-            NewPipe.init(DownloaderTestImpl.getInstance());
+            final File file = new File(RESOURCE_PATH + "mix");
+            NewPipe.init(new DownloaderFactory().getDownloader(file.getAbsolutePath()));
```

Comment:
Maybe you could also add a method to DownloaderFactory replacing these two lines, so that they don't have to be repeated every time. Something like `DownloaderFactory.initNewPipeWithDownloader(RESOURCE_PATH + "mix")`, and put the `.getAbsolutePath()` inside there, too.

Reply:
I found that creating the file object itself is unnecessary. without that line its already a single liner.